api/app/src/main/kotlin/packit/service/UserService.kt

```diff
@@ -47,6 +51,41 @@ class BaseUserService(
         return newUser
     }
 
+    override fun createBasicUser(createBasicUser: CreateBasicUser)
+    {
+        val existingUser = userRepository.findByUsername(createBasicUser.email)
```

Comment:
See the comment in previous PR - I think the username should be the primary identifier for all user types - though in the case of basic users those will always be the same anyway!